README.md

```diff
@@ -203,7 +110,42 @@ html_context = {
 
 By enabling analytics we will be able to collect information on number of visits to each documentation page. It will also trigger the addition of a `Was this page helpful?` component at the bottom of each documentation page, so users will be able to provide yes/no feedback for each page.
 
-<img width="538" alt="Screenshot 2022-11-14 at 11 28 11 AM" src="https://user-images.githubusercontent.com/23662430/201715694-1e75b1fb-875b-4137-b9f3-93d1e1894f5a.png">
+![](tests/js/snapshots.test.js-snapshots/footer-includes-page-analytics-1-linux.png)
+
+If you do not enable analytics, no data will be collected and the `Was this page helpful?` component will not show.
+
+## Migrate from old Pytorch theme to new theme
+
+In qiskit-sphinx-theme 1.13, we migrated to a new Sphinx theme based on Furo, which is used by pip, Black, and attrs documentation. See https://github.com/Qiskit/qiskit_sphinx_theme/issues/232 for the motivation.
+
+qiskit-sphinx-theme 1.13 continues to support the legacy Pytorch theme, but support will be removed in version 2.0.
+
+To migrate:
+
+1. In `conf.py`, set `html_theme = "qiskit"` rather than `"qiskit_sphinx_theme"`.
+2. In `conf.py`, remove all `html_theme_options`.
+3. In `conf.py`, remove `expandable_sidebar` from `html_context`, if set.
+4. If `expandable_sidebar` was enabled, then update your `index.rst` files. The new theme always has expandable sidebars. Before, you had to have a dedicated `.. toctree::` directive for each expandable folder, along with the `:caption:` option set to the folder name. To migrate, consolidate all of your `.. toctree::` directives into one. Alternatively, you can keep the separate `.. toctree::` entries; the `:caption:` will render as a top-level header and the folder contents will be expanded.
```

Comment:
I find this section a bit difficult to follow, especially when it comes to the exapandable sidebar stuff. Could you add a code example of how one would have their .rst structure in the old format and compare it to the new format. Or maybe add a whole new section explaining how Furo handles the sidebar stuff? That might be better actually for people who are coming to the theme for the first time with a new project

Reply:
Honestly, I'm still trying to figure out exactly how the migration is going to work. Would it be okay to improve this in a follow-up PR once I've migrated some projects?

Also, I will be the one doing all these migrations, so these docs are mostly to be diligent / best practice.